### PR TITLE
[Test] Start on notebook testing

### PR DESCRIPTION
**Pull request by riedgar-ms**
Begin working on testing notebooks. This just gets two working, since others are being.... problematic. The tests also don't do anything beyond make sure that the notebook runs without throwing exceptions.

### CODE REVIEW

**Comment by riedgar-ms** [comment]
@Harsha-Nori are you OK with this?

**Comment by riedgar-ms** [comment]
In the interests of not getting too backlogged, I propose to merge this tomorrow (26th) unless there are objections.

File: tests/test_notebooks.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+BASE_NB_PATH = pathlib.Path("./notebooks").absolute()
+
+
+def run_notebook(notebook_path: pathlib.Path):
+    output_nb = notebook_path.stem + ".papermill_out" + notebook_path.suffix
+    output_path = TestTutorials.BASE_TUTORIAL_PATH / output_nb
+
+    # Just make sure nothing throws an exception
+    pm.execute_notebook(input_path=notebook_path, output_path=output_path)
```
Comment (Harsha-Nori) [inline tests/test_notebooks.py:13]: Seems like a reasonable place to start

File: setup.py [hunk 2]
```diff
@@ -41,23 +48,20 @@ def find_version(*file_paths):
...
-            'torch',
-            'pytest-cov'
-        ]
+            "jupyter",
+            "papermill",
+            "pytest",
+            "pytest-cov",
+            "transformers",
+            "torch",
+            "transformers",
```
Comment (Harsha-Nori) [inline setup.py:64]: let's keep this and drop the other `black` format changes to setup.py for now (completely OK with these testing dependencies being added)

**Pull request merged**